debian/conf/jellyfin.init

```diff
@@ -19,13 +19,14 @@ fi
 
 . /lib/lsb/init-functions
 
+JELLYFIN_HOME="/etc/default/jellyfin"
```

Comment:
Oh that's not what I meant here.

Basically, there should already be a variable inside that file that defines what the data directory is (`JELLYFIN_DATA_DIR`) so that should be used for the `--chdir` below instead. Basically no hardcoding of paths inside of here (PIDfile and such notwithstanding).

Reply:
Check if what i made is correct or not